pkg/occlient/occlient.go

```diff
@@ -1150,13 +1173,44 @@ func (c *Client) GetBuildConfig(name string, project string) (*buildv1.BuildConf
 // GetClusterServiceClasses queries the service service catalog to get all the
 // currently available cluster service classes
 func (c *Client) GetClusterServiceClasses() ([]scv1beta1.ClusterServiceClass, error) {
```

Comment:
Thi is is a bit confusing name and commend. It says that you are getting all the cluster services, but  in listing you limit it to just template service broker